api-reference/api-reference-template.md

```diff
@@ -0,0 +1,204 @@
+# API Reference
```

Comment:
Shouldn't this line be a variable for the actual reference page's title? Something like: 
# {page title, ex: "Send a message" or "POST /message"}

Reply:
@csmeyns I think this template is meant for people manually writing an API reference page, rather than generating it from a spec. Not everyone has the expertise or toolset in their project to work from OAS or similar. But we should probably at least mention the OAS option as an alternative method for generating the page, maybe at the very beginning of the guide page.